lib/chunks.js

```diff
@@ -458,7 +465,7 @@ module.exports.diffChunks = async ({ coursePath, courseId, courseData, changedFi
         (!existingCourseChunks[chunkType] || changedCourseChunks[chunkType])
       ) {
         updatedChunks.push({ type: chunkType });
-      } else if (!hasChunkDirectory) {
+      } else if (!hasChunkDirectory && existingCourseChunks[chunkType]) {
```

Comment:
This was an interesting one: we were marking these chunks as needing deletion even if they didn't exist! In practice, this wasn't actually a bug, as Postgres silently ignores these (as it should). However, it would be surprising for instructors if these showed up in logs, so I fixed this.

Reply:
Good catch!